templates/base.jinja2

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Karen Wong | Learning Journal</title>
+    <link rel="stylesheet" href="css/normalize.css">
+    <!--[if lt IE 9]>
+    <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+    <![endif]-->
+    <link href="/static/style.css" rel="stylesheet" type="text/css">
+    <meta name="viewport" content="width=device-width, initial-scale-1.0">
+  </head>
+  <body>
+    <div id="logo">
+        <img src="/static/img/logo.jpeg" alt="The Learning Journal" />
+     </div>
+    <header>
+        <nav>
+            <ul>
+                <li><a href='{{ request.route_url('home') }}'
+                  class="selected">+Home</a></li>
+                <li><a href='{{ request.route_url('add') }}'>+New Entry</a>
+                </li>
+                <li><a href='{{ request.route_url('detail') }}'>+View Latest Entry</a></li>
```

Comment:
This doesn't really show you the latest entry.  It shows you a list of all the entries.